addon-test-support/should-force-audit.ts

```diff
@@ -0,0 +1,13 @@
+const _url: URL = new URL(window.location.href, document.baseURI);
+
+export function setEnableA11yAudit(enabled: boolean = false) {
+  if (enabled) {
+    _url.searchParams.set('enableA11yAudit', 'true');
+  } else {
+    _url.searchParams.delete('enableA11yAudit');
+  }
+}
+
+export function shouldForceAudit() {
+  return _url.searchParams.get('enableA11yAudit') !== null;
+}
```

Comment:
I think this is static, and won't work. The `_url` variable won't live update when the running URL bar changes...

```suggestion
export function shouldForceAudit() {
  const url = new URL(window.location.href, document.baseURI);

  return url.searchParams.get('enableA11yAudit') !== null;
}
```